Storage/StorageTests/CoreData/MigrationTests.swift

```diff
@@ -2102,6 +2102,26 @@ final class MigrationTests: XCTestCase {
         // OrderItem's addOns relationship exists.
         XCTAssertEqual(migratedOrderItem.value(forKey: "addOns") as? NSOrderedSet, [addOn])
     }
+
+    func test_migrating_from_94_to_95_enables_creating_new_TaxRate_entity() throws {
+        // Given
+        let sourceContainer = try startPersistentContainer("Model 94")
+        let sourceContext = sourceContainer.viewContext
+
+        try sourceContext.save()
+
+        // When
+        let targetContainer = try migrate(sourceContainer, to: "Model 95")
+
+        // Then
+        let targetContext = targetContainer.viewContext
+        XCTAssertEqual(try targetContext.count(entityName: "TaxRate"), 0)
+
+        _ = insertTaxRate(to: targetContext)
```

Comment:
As above, we should test as well the existance/non-existance of these properties just to be sure:

```suggestion
        let taxRate = insertTaxRate(to: targetContext)
        XCTAssertEqual(taxRate.value(forKey: "city") as? String, "Miami")
        // etc, .. 
```